corehq/ex-submodules/couchexport/export.py

```diff
@@ -309,7 +309,8 @@ def split_path(path):
         id = []
         for k in path:
             if isinstance(k, basestring):
-                column.append(k)
+                if k:
+                    column.append(k)
```

Comment:
ah ok cool i think i understand. i think it's worth going for. in the worst case, the amount of users it will affect if it doesn't work is pretty small